package-lock.json

```diff
@@ -2432,29 +9916,25 @@
         },
         "deep-extend": {
           "version": "0.6.0",
-          "resolved": false,
-          "integrity": "sha512-LOHxIOaPYdHlJRtCQfDIVZtfw/ufM8+rVj649RIHzcm/vGwQRXFt6OPqIFWsm2XEMrNIEtWR64sY1LEKD2vAOA==",
+          "bundled": true,
           "dev": true,
           "optional": true
         },
         "delegates": {
           "version": "1.0.0",
-          "resolved": false,
-          "integrity": "sha1-hMbhWbgZBP3KWaDvRM2HDTElD5o=",
+          "bundled": true,
```

Comment:
I think those changes are due to a new npm version on your machine. I'm not sure what npm version we use and wether this is fine or not. (May be it will break the CI or something like that 🤷 )

Reply:
As far as I read, the lockfile version 2 is backwards compatible.

Reply:
Actually we don't have CI yet :)